src/util/moneystr.cpp

```diff
@@ -37,7 +37,7 @@ bool ParseMoney(const std::string& str, CAmount& nRet)
         return false;
     }
 
-    if (str.empty()) {
+    if (TrimString(str, " ").empty()) {
```

Comment:
Why not the default pattern, which covers other whitespace?

Reply:
Only spaces are handled in the existing code and I want to keep the changes to the minimum required to make the function consistent in its handling of spaces (which is the only whitespace considered).